doc/user_guide/dynamo0p3.rst

```diff
@@ -3754,22 +3754,22 @@ values are defined in the LFRic infrastructure code.
 Precision Map
 +++++++++++++
 
-When scalars are defined in a kernel, they are allocated a set amount of
-memory. This amount, given in bytes, is currently defined in the precision
-map within the LFRic API *and* the LFRic infrastructure.
+The amount of storage (in bytes) associated with a particular LFRic precision
+is currently defined in a kernel, they are allocated a set amount of memory.
```

Comment:
I'm not sure what you're saying here? The amount of storage is not defined in a kernel. This (L3757-3760) needs to say something like:

ThIs specifies the amount of storage (in bytes) that is associated with each LFRic precision.